nginx/setup.sh

```diff
@@ -42,5 +41,18 @@ export HULY_SECRET="c37fef1f157efe09785844215225ee3d1ab8086eb822055d5aa1bdedc729
 # replace the domain name and email address in the docker-compose file
 envsubst < template-compose.yml > docker-compose.yml
 
-echo -e "\033[1;32mSetup is complete. Run 'docker compose up -d' to start the services.\033[0m"
+echo -e "\033[1;32mSetup is complete!\033[0m"
 
+read -p "Do you want to run 'docker compose up -d' now to spin up Huly? ([Y]es/[n]o): " RUN_DOCKER
+case "${RUN_DOCKER,,}" in  
```

Comment:
This does not work for me. But I'm using zsh that has different syntax for lower case. I suggest to simplify options in the `case` to `[yY]` and `[nN]`. What do you think?